internal/services/openai.go

```diff
@@ -89,16 +89,16 @@ func (o openAI) askChatGPT(body io.Reader) (*ChatGPTResponse, error) {
 	return cResp, nil
 }
 
-func (o openAI) GetErrorCodesDescription(make, model string, year int32, errorCodes []string) (string, error) {
+func (o openAI) GetErrorCodesDescription(make, model string, errorCodes []string) (string, error) {
 	codes := strings.Join(errorCodes, ", ")
 	req := fmt.Sprintf(`{
 		"model": "gpt-3.5-turbo",
 		"messages": [
 			{
 				"role": "user", 
-				"content": "Briefly summarize for me, in order, what the following error codes mean for a %s %s %d. The error codes are %s."}]
+				"content": "Briefly summarize for me, in order, what the following error codes mean for a %s %s. The error codes are %s."}]
```

Comment:
I could have sworn I fixed up the formatting here. I guess it got undone by the second PR.

Reply:
More importantly: the last `%s` might be replaced by any characters. `"`, `"}`, etc. It's not the worst hole in the world but it seems needless.